CMakeLists.txt

```diff
@@ -1,15 +1,44 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.3)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+
+INCLUDE(GNUInstallDirs)
 
-# Name & Version
 PROJECT(LLVM_SPIRV
-        VERSION   0.1.0.0
-        LANGUAGES CXX)
+  VERSION
+    0.1.0.0
+  LANGUAGES
+    CXX)
+
+SET(CMAKE_CXX_STANDARD 11)
+SET(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+FIND_PACKAGE(LLVM 7.0.0 REQUIRED)
```

Comment:
I think we should look for installed LLVM only if we build out-of-tree. Could you add a check for that similar to how clang does it?

Reply:
I could just copy what clang does, but I'm not quite sure if it's worth to care about it for now, because at least ll distributions packaging this wouldn't build it this way. And the normal user doesn't want to clone llvm just to build this tool either.

So for what kind of users would we want to support building it with llvm together?

Reply:
For example for developers who already have llvm and clang sources cloned. They likely to have some modifications in clang or llvm sources. They as well may have custom installation directory. They might want to test how their changes in clang affect the translator. In this case it is easier to build and test everything at once.